src/main/java/com/google/sps/servlets/schedule/UserPreferences.java

```diff
@@ -38,30 +43,54 @@
-    public Map<Integer, Integer> STUDY_SESSION_START_TIME = new LinkedHashMap<Integer, Integer>();
-    public Map<Integer, Integer> STUDY_SESSION_LENGTH = new LinkedHashMap<Integer, Integer>();
+    public static Map<Integer, Integer> STUDY_SESSION_START_TIME = new LinkedHashMap<Integer, Integer>() {{
+        put(9, 0); // 9AM
+        put(12, 0); // 12AM
+        put(16, 0); // 4PM
+        put(19, 0); // 7PM
+    }};
+
+    public static Map<Integer, Integer> STUDY_SESSION_LENGTH = new LinkedHashMap<Integer, Integer>() {{
+        put(1, 0); // 1 hour
+        put(0, 30); // 30 minutes
+    }};
 
-    // Set description if no resources could be found
-    public static String DESCRIPTION = "Read for motivation: https://medium.com/dev-genius/why-you-should-stop-watching-tutorials-and-reading-docs-escaping-tutorial-purgatory-2c6b438b418 \n\n Best recommended free site: https://www.codecademy.com/\n\nBest recommended paid site: https://www.udemy.com/";
-    
     // Fixer Function method of correcting events
     public static enum FixerCorrection { DELETE_EVENT, FIND_NEXT_AVAIL_TIME, FORCE_MOVE_TO_DAY, LEAVE_AS_IS };
     public static FixerCorrection FIXER_CORRECTION_CHOICE = FixerCorrection.FIND_NEXT_AVAIL_TIME;
 
-    // Set time points of where we will try to schedule an event -- 24HR TIME
-    public void applyStudySessionStartTimes() { 
-        STUDY_SESSION_START_TIME.put(9, 0); // 9AM
-        STUDY_SESSION_START_TIME.put(12, 0); // 12PM
-        STUDY_SESSION_START_TIME.put(16, 0); // 4PM
-        STUDY_SESSION_START_TIME.put(19, 0); // 7PM
-    }
-    
+    // Set description if no resources could be found or out of resources
+    public static String DESCRIPTION = new StringBuilder(
+       "Need some motivation: https://medium.com/dev-genius/why-you-should-stop-watching-tutorials-and-reading-docs-escaping-tutorial-purgatory-2c6b438b418\n\n" +
+       "Looking for extra free material?: https://www.codecademy.com/\nhttps://www.khanacademy.org/\n\n" + 
+       "Eager to pay for material?: https://www.udemy.com/\n\n" +
+       "Interview Prep?: https://leetcode.com/problemset/all/\nhttps://www.algoexpert.io/product").toString();
 
-    // Set Durations Of Our Study Sessions
-    public void applyDurationStartTimes() {
-        // hours, minutes
-        STUDY_SESSION_LENGTH.put(1, 0); // 1 hour
-        STUDY_SESSION_LENGTH.put(0, 30); // 30 minutes
+    public static List<Integer> getPossibleDays() {
+        List<Integer> days = new ArrayList<Integer>();
+        switch(STUDY_SESSION_DAYS_CHOICE) {
+            case CUSTOM:
+                days = STUDY_SESSION_CUSTOM_DAYS;
+                break;
+            case WEEKDAY:
+                days = new ArrayList<Integer>() {{
+                    add(1);
+                    add(2);
+                    add(3);
+                    add(4);
+                    add(5); // less than or equal to 5 according to JodaTime is the weekday
+                }};
```

Comment:
new ArrayList<Integer>(Arrays.asList(1, 2, 3, 4, 5));

https://www.java67.com/2015/10/how-to-declare-arraylist-with-values-in-java.html

Reply:
Right!